### DISCUSSION TITLE
Error when running an example online

### DISCUSSION THREAD

**Discussion by agenceKanvas** [visgl/deck.gl#8092] [General]
Hi !
I've set up my server locally with a custom url. 
Everything is working (php running). I've cloned the repo, gone to the website heatmap exemple, run npm install and npm start, no error.
Problem is, when I visit the page, not with the IP but with the local URL, i have this error : 
```Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "". Strict MIME type checking is enforced for module scripts per HTML spec.```
What am I missing ? 
I'm working on mac, with Local server.
Thanks !

**Comment by Pessimistress** [comment]
As the error suggests, your server is not serving the file with the correct MIME type (`application/javascript`).
